plugins/Goals/vue/src/ManageGoals/ManageGoals.vue

```diff
@@ -86,21 +85,17 @@
 
                 <td v-if="userCanEditGoals" style="padding-top:2px">
                   <button
+                    v-if="userCanEditGoals"
```

Comment:
The v-if on both buttons is superfluous as the whole column already has the same condition.

Reply:
True. I figured I'd leave it there in case a new action is added and the td's v-if condition changes.